mount/fake_mounter.go

```diff
@@ -117,7 +126,7 @@ func (f *FakeMounter) Mount(source string, target string, fstype string, options
 	if err != nil {
 		absTarget = target
 	}
-	f.MountPoints = append(f.MountPoints, MountPoint{Device: source, Path: absTarget, Type: fstype, Opts: opts})
+	f.MountPoints = append(f.MountPoints, MountPoint{Device: source, Path: absTarget, Type: fstype, Opts: append(opts, sensitiveOptions...)})
```

Comment:
are options ever order sensitive, or can sensitive options always be grouped together at the end?

Reply:
Ceph appends them to the end https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/cephfs/cephfs.go#L315

I'll check Azure

Reply:
AzureFile also appears to tack them on to the end 
https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/azure_file/azure_file.go#L284

Reply:
add a comment to the MountPoint#Opts field that indicates it may contain sensitive values and should not be logged?

Reply:
Done.